tox.ini

```diff
@@ -1,59 +0,0 @@
-# Copyright (c) 2020 Hong Xu <hong@topbug.net>
```

Comment:
The reason that I had this tox.ini here is to avoid overdepending on a particular CI system. The appveyor file is pretty minimal and the heavy work is done in this tox.ini file.

Reply:
Understandable. With all linting wrapped in pre-commit the Github Actions workflow is also relatively simple and could be easily ported to another CI. We could still wrap them in tox if you prefer.